apps/teams-test-app/e2e-test-data/location.json

```diff
@@ -19,6 +20,7 @@
     {
       "title": "showLocation API Call - Success",
       "type": "callResponse",
+      "platformsExcluded": ["iOS"],
```

Comment:
Just curious, why we will exclude ios platform on this one? As I see we exclude the web platform on "getLocation API Call - Success"

Reply:
Updated in description and please feel free to let me know if it makes sense.

Reply:
LGTM and approved from my side. Would be great to include more approver from whoever familiar with teamsjs codebase.

Reply:
Do we need to exclude android as well?

Reply:
Ideally yes, but for now, Android doesn't support back-compat so we don't care. There will be another person update here in the future for Android.